Gems/ScriptCanvasDeveloper/Code/CMakeLists.txt

```diff
@@ -22,12 +22,15 @@ ly_add_target(
     COMPILE_DEFINITIONS
         PRIVATE
             ${SCRIPT_CANVAS_DEV_COMMON_DEFINES}
+            IMGUI_ENABLED
     BUILD_DEPENDENCIES
```

Comment:
Question: as we enable it as build dependency, do we have anything in developer gem that requires to be built based on ImGui?

Reply:
Yes, there is a performance reporting feature in the gem's system component that uses ImGui